### ISSUE TITLE
Cannot Connect to hostname MongoDB Atlas - MONGODB_SRV

### ISSUE THREAD

**Issue by sapisuper** [kviklet/kviklet#217]
image version : main 
Database Protocol : MONGODB_SRV 

Hi, 

I just tried to add connection MongoDB (not using ip) but using MongoDB Atlas endpoint. but got this error : 
```
Host for when using mongodb+srv protocol can not contain a port
```

So I remove port value in the Advance Configuration. but got different error: 
```
The connection string contains an invalid host 'xxxx.obuqg.mongodb.net:0'. The port '0' is not a valid, it must be an integer between 0 and 65535
```
the port return 0 even I deleted the value 

Detail screencapture 
![Screenshot 2024-08-28 at 18 37 22](https://github.com/user-attachments/assets/361620ee-75ea-4d76-9352-d001e981aeb0)

**Comment by Askir** [comment]
Woops, good catch! Fixed in in #218

**Comment by sapisuper** [comment]
@Askir tested work well, thank you 👍 
![Screenshot 2024-08-29 at 09 52 34](https://github.com/user-attachments/assets/9af58b0c-260a-4cce-8b6c-cf3dca9a4095)
